lib/samples/set_basemap/set_basemap.dart

```diff
@@ -124,7 +124,7 @@ class _SetBasemapState extends State<SetBasemap> with SampleStateSupport {
             child: FloatingActionButton(
               onPressed: () => _scaffoldStateKey.currentState!.openEndDrawer(),
               shape: const RoundedRectangleBorder(),
-              backgroundColor: Colors.deepPurple,
+              backgroundColor: Theme.of(context).primaryColor,
```

Comment:
I think we can use `ThemeData.floatingActionButtonTheme` to specify both `shape` and `backgroundColor` for all `FloatingActionButton`s. Then we don't have to specify these properties for every `FloatingActionButton`.